README.md

```diff
@@ -29,6 +29,13 @@ add the option `withKeepAlive` on the environment (`new SeleniumJSEnv(BROWSER).w
 It is recommend to use this with a `run` and not `test` because the latter tends
 to leave too many browser windows open.
 
+#### Debugging tests on a single window
+By default tests are executed in their on window for isolation and parallelism.
```

Comment:
~~for isolation~~
That's not true. We run many tests in the same window. It's only for parallelism.